tools/generate-sources-manifests.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+SOURCES_PATH=$1
+MANIFESTS_FILE=$2
+rm -f $MANIFESTS_FILE
+echo $SOURCES_PATH
+BASEDIR="$(dirname "${SOURCES_PATH}")"
+echo $BASEDIR
+tar -xf $SOURCES_PATH -C $BASEDIR
+## alpine packages
+alpine_manifests() {
+cd $BASEDIR/alpine
+for file in `ls`
+do
+        commit=$(echo $file| rev | cut -d "." -f1 | rev)
+        name_version=$(echo $file | rev | cut -d "." -f2- | rev )
+        echo "alpine",$name_version,$commit,"alpine/$file" >> $MANIFESTS_FILE
+done
+}
+
+## go packages
+
+go_manifests() {
+cd $BASEDIR/go
```

Comment:
This cd approach implicitly assumes that $1 hence BASEDIR is an absolute path, which I'm not sure the Makefile caller guarantees.

Reply:
This is fixed.